docs/configuration-guide.md

```diff
@@ -617,7 +617,7 @@ The section below describes all the available subtypes.
 | dekorate.certificate.commonName       | String          | common name to be used on the Certificate                 |                      |  
 | dekorate.certificate.duration         | String          | the lifetime of the Certificate                           |                      |  
 | dekorate.certificate.renewBefore      | String          | the renewal before duration timestamp                     |                      |  
-| dekorate.certificate.dnsNames         | String[]        | the list of DNS subjectAltNames                           |                      |  
+| dekorate.certificate.dnsNames         | String[]        | the list of [Subject Alternative Names](https://en.wikipedia.org/wiki/Subject_Alternative_Name) that should be included as part of the Order validation process                           |                      |  
```

Comment:
Having that link is a great plus but it will be needed, and perhaps within the documentation, that we give some examples as the FQNames should match the URL of the Services exposed [externally](https://kubernetes.io/docs/concepts/services-networking/ingress/) or [accessed internally](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#a-aaaa-records).

Reply:
I don't think it's our responsibility to explain how Cert-Manager works. Maybe, adding a link to Cert-Manager here is enough.

Reply:
This is our responsibility to avoid that users using dekorate will struggle with HTTPS Certificate error AND will spend hours to figure out what they should do (example: pass the appropriate ingress address or openshift route) :-)

Reply:
I've added a new section about this (I've included another link here).